user_modules/utils.lua

```diff
@@ -0,0 +1,97 @@
+local module = {}
+
+local common_functions = require("user_modules/common_functions")
```

Comment:
@VVeremjova 
Please correct path to common_functions module (user_modules/shared_testcases/commonFunctions.lua)